app/routes/users/components/PhotoConsents.js

```diff
@@ -0,0 +1,182 @@
+// @flow
+
+import { useState } from 'react';
+import Select from 'react-select';
+import Button from 'app/components/Button';
+import Flex from 'app/components/Layout/Flex';
+import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
+import type { PhotoConsent, PhotoConsentDomain } from 'app/models';
+import moment from 'moment-timezone';
+import cx from 'classnames';
+import {
+  PHOTO_CONSENT_DOMAINS,
+  toReadableSemester,
+} from 'app/routes/events/utils';
+import styles from './PhotoConsents.css';
+
+const ConsentManager = ({
+  consent,
+  updateConsent,
+  isMe,
+}: {
+  consent: ?PhotoConsent,
+  updateConsent: (consent: PhotoConsent) => Promise<*>,
+  isMe: boolean,
+}) => {
+  if (!consent) {
+    return null;
+  }
+
+  const getConsentStatus = () => {
+    if (consent.isConsenting) {
+      return (
+        <>
+          <b>Du ga samtykket den </b>
+          <i>{moment(consent.updatedAt).format('DD. MMM YYYY')}.</i>
+        </>
+      );
+    } else if (consent.isConsenting === false) {
+      return (
+        <>
+          <b>Du trakk samtykket den </b>
+          <i>{moment(consent.updatedAt).format('DD. MMM YYYY')}.</i>
+        </>
+      );
+    }
+    return <b>Du har ikke tatt stilling til samtykket.</b>;
+  };
+
+  const presentableDomain =
+    consent.domain === PHOTO_CONSENT_DOMAINS.WEBSITE
+      ? 'Abakus.no'
+      : 'Sosiale medier';
+  return (
+    <>
+      <h4 className={styles.categoryTitle}>{presentableDomain}</h4>
+      <h5>
+        Jeg godtar at Abakus kan legge ut bilder av meg på {presentableDomain} i
+        perioden {toReadableSemester(consent)}:
+      </h5>
+      <div className={styles.statusContainer}>{getConsentStatus()}</div>
+      <div>
+        <ConfirmModalWithParent
+          closeOnConfirm={true}
+          title={`Trekke bildesamtykke på ${presentableDomain}`}
+          message={`Er du sikker på at du vil trekke bildesamtykket ditt for ${toReadableSemester(
+            consent
+          )} på ${presentableDomain}? Dersom du ønsker å fjerne noen spesifikke bilder, kan du i stedet sende en mail til pr@abakus.no med informasjon om hvilke bilder du vil fjerne.`}
+          onConfirm={() => updateConsent({ ...consent, isConsenting: false })}
+        >
+          <Button
+            disabled={!isMe}
+            className={
+              consent.isConsenting === false ? styles.selectedConsentBtn : ''
+            }
+          >
+            Nei
+          </Button>
+        </ConfirmModalWithParent>
+        <Button
+          disabled={!isMe}
+          onClick={() => updateConsent({ ...consent, isConsenting: true })}
+          className={
+            consent.isConsenting
+              ? cx(styles.consentBtn, styles.selectedConsentBtn)
+              : styles.consentBtn
+          }
+        >
+          Ja
+        </Button>
```

Comment:
Could you rebase this and employ the new `dark` and `success` buttons?

Reply:
And, btw, can "Ja" come before "Nei"? I feel like that's better mapping

Reply:
Actually, another point here is to not use just "Ja" and "Nei" and instead label the buttons with the actions they perform. I.e. use "Trekk samtykke" instead of "Ja" and "Behold samtykke" instead of "Nei".

Reply:
Refactored to the sexy new buttons and changed the text to "Behold samtykke" + "Trekk samtykke" ✌🏻